demos/react-jsnation/tests/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "private": true,
+  "type": "module",
+  "name": "@starbeam-demos/jsnation/tests",
```

Comment:
I'm open to a different naming convention here. I know you've used `@starbeam-tests` as the root for some things, but I wasn't sure if the demo tests would fall under that classification. Let me know what name you want and I can change it, no problem.